_commontech/Customer_3.0.0_Release_Notes.md

```diff
@@ -8,45 +8,45 @@ dateAdded: April 22, 2016
 author: Leslie Lundquist
 ---
 
-## IBM Blue Box Cloud Customer
+## IBM Bluemix on OpenStack Customer
 
 ## Release Notes 3.0.0
 
 #### June 30, 2016
 
 
-This document offers a basic technical description of the latest IBM Blue Box Cloud offerings, including specifics of the 3.0 Dedicated and Local Cloud offerings. Links to detailed documentation can be found in each section.
+This document offers a basic technical description of the latest IBM Bluemix on OpenStack offerings, including specifics of the 3.0 Dedicated and Local Cloud offerings. Links to detailed documentation can be found in each section.
 
 ### Summary of the Latest Release
 
-Release 3.0.0 of IBM Blue Box Cloud includes many updates to the underlying OpenStack platform upon which every IBM Blue Box Cloud is built, as well as upgrades to infrastructure and features of IBM Blue Box Dedicated Cloud and IBM Blue Box Local Cloud. Most notable is the move to the **OpenStack Mitaka** release, which enables many new capabilities. We’ve included some general product improvements such as performance benchmarks and an updated Rally test suite, designed to facilitate smoother operation and deployment. This release also includes many improvements to documentation.
+Release 3.0.0 of IBM Bluemix on OpenStack includes many updates to the underlying OpenStack platform upon which every IBM Bluemix on OpenStack is built, as well as upgrades to infrastructure and features of IBM Bluemix Dedicated on OpenStack and IBM Bluemix Local on OpenStack. Most notable is the move to the **OpenStack Mitaka** release, which enables many new capabilities. We’ve included some general product improvements such as performance benchmarks and an updated Rally test suite, designed to facilitate smoother operation and deployment. This release also includes many improvements to documentation.
```

Comment:
Please double check the comment from PR #421 for this line, line 22. This paragraph needs to be reworded now that the name has changed.